realm/realm-library/src/main/cpp/jni_util/java_class.cpp

```diff
@@ -47,7 +47,7 @@ JavaClass::JavaClass(JavaClass&& rhs)
 JavaGlobalRef JavaClass::get_jclass(JNIEnv* env, const char* class_name)
 {
     jclass cls = env->FindClass(class_name);
-    REALM_ASSERT_RELEASE(cls);
+    REALM_ASSERT_RELEASE(class_name);
```

Comment:
This actually changes the semantics of the assertion. we want to check if the class can be found or not.

`REALM_ASSERT_RELEASE_EX` should be used in this case.

I am checking the original issue now.

Reply:
Ups, you are right. The idea was that I wanted to report what class was missing if it wasn't present